src/vue/pages/StatisticsGeneral.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="statistics-general">
+    <statistics-filters
+      @set-filters-and-update-list="setFiltersAndUpdateList"
+      @show-no-assets-message="showNoAssetsMessage = true"
```

Comment:
useless, you can check availability assets here( assets are stored in store )

Reply:
removed